### PR TITLE
Homework 2 project files

### PR DESCRIPTION
**Pull request by AnduAnches**

### CODE REVIEW

File: homework_2/.idea/compiler.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (MelindaFancsal) [inline homework_2/.idea/compiler.xml:1]: the entire .idea/ folder should be gitignored. Please add the .gitignore file and include:
+*.idea
 +*.iml
 +*.ims
 +*.ipr
 +*.class
 +*.zip

File: homework_2/homework_2.iml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (MelindaFancsal) [inline homework_2/homework_2.iml:1]: as mentions ./idea files should not be commited

File: homework_2/src/main/java/ro/siit/main.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+ * @version 1.0
+ * @since 2018-05-20
+ */
+
+package ro.siit;
+import java.util.Scanner;
+
+public class main {
+    public static void main(String[] args) {
+        /**
```
Comment (MelindaFancsal) [inline homework_2/src/main/java/ro/siit/main.java]: I like the fact that you add documention here for the methods in classes, but I don t think that here we need to add it.

**Review by MelindaFancsal — APPROVED** [review]

File: homework_2/.idea/compiler.xml (hunk 1 above)
Comment (AnduAnches) [reply homework_2/.idea/compiler.xml:1]: Hello, thanks for the feedback, I understood wrong, I thought the whole project must be added to the repo. I'll do the changes suggested.

File: homework_2/src/main/java/ro/siit/main.java (hunk 3 above)
Comment (AnduAnches) [reply homework_2/src/main/java/ro/siit/main.java]: Removed the methods' documentation, thanks!

**Pull request merged**